db.json

```diff
@@ -1 +1 @@
-{"pw":" Maus"}
\ No newline at end of file
+{ "pw": "ae70a97f8f0b2b3c0a6838edcb6584fb" }
```

Comment:
your db.json has to be gitignored. you can use `git rm --cached <filename>` for removing it from your remote repo.